dev/com.ibm.websphere.appserver.features/visibility/public/xmlWS-3.0/io.openliberty.xmlWS-3.0.feature

```diff
@@ -10,7 +10,6 @@ IBM-API-Package: \
  org.apache.cxf.binding.soap.wsdl.extensions;type="internal", \
  org.apache.cxf.databinding;type="internal"
 -features=\
- com.ibm.websphere.appserver.eeCompatible-9.0, \
```

Comment:
Dropped this down into the xmlws.common-3.0 feature so that can be added to the xmlWSClient-3.0 feature as well.